scripts/addSettingsMissingFields.js

```diff
@@ -0,0 +1,12 @@
+require('dotenv').load({silent: true});
```

Comment:
Very useful script, thanks!

Reply:
We could even try and use it on server startup, let's leave that for future sprints.